configs/nginx/timenow.local

```diff
@@ -0,0 +1,23 @@
+server {
+	listen 443; 
+	listen [::]:443;
+
+	root /var/www/timenow; 
+
+	server_name timenow.local;
+
+	location /api/ { 
+		proxy_pass http://localhost:8080/;
+		add_header Access-Control-Allow-Origin http://timenow.local/api;
```

Comment:
The URL should not include the path (/api). Instead, you should use the full origin (http://timenow.local)